src/status_im/ui/screens/discover/subs.cljs

```diff
@@ -2,18 +2,33 @@
   (:require [re-frame.core :refer [reg-sub]]
             [status-im.utils.datetime :as time]))
 
-(defn- calculate-priority [{:keys [chats current-public-key]
-                            :contacts/keys [contacts]}
+(reg-sub :get-discoveries
+  (fn [db]
+    (:discoveries db)))
+
+(reg-sub :get-current-tag
+  (fn [db]
+    (:current-tag db)))
+
+(reg-sub :get-discover-search-tags
+  (fn [db]
+    (:discover-search-tags db)))
+
+(reg-sub :get-tags
+  (fn [db]
+    (:tags db)))
+
+(defn- calculate-priority [chats current-public-key contacts
```

Comment:
Do we have tests for this?

Reply:
No we don't, but it's a good opportunity to cover it with tests.